examples/L5PC/README.rst

```diff
@@ -345,4 +345,12 @@ To address this, you can include the following line in your sbatch files to set
 
     export NEURON_MODULE_OPTIONS="-nogui"
 
-This line is intended to prevent NEURON from sending any GUI info. An alternative solution would be to disable X11 forwarding altogether in your SSH session.
\ No newline at end of file
+This line is intended to prevent NEURON from sending any GUI info. An alternative solution would be to disable X11 forwarding altogether in your SSH session.
+
+Issues with ipyparallel
+~~~~~~~~~~~~~~~~~~~~~~~
+Sometimes, you may encounter issues with ipyparallel during optimisation. If this happens, you can switch to using multiprocessing instead. To do this, modify the pipeline.py command in the optimisation.sbatch script as follows:
+
+.. code-block:: shell
+
+    python pipeline.py --use_multiprocessing --step='optimise' --emodel=${OPT_EMODEL} --seed=${OPT_SEED} --githash=${GITHASH}
```

Comment:
could you tell here that the users will also have to remove the ipyparallel related [lines](https://github.com/BlueBrain/BluePyEModel/blob/64cfffcf4988d7e7f8daf664f3c1d25a75396e43/examples/L5PC/optimisation.sbatch#L33) from the *.sbatch files:

```
export IPYTHON_PROFILE=extract_${SLURM_JOB_ID}_$(hostname)
export USEIPYP=1
export IPYTHONDIR="`pwd`/.ipython"

ipcontroller --init --ip='*' --profile=${IPYTHON_PROFILE} &
sleep 20
srun ipengine --profile=${IPYTHON_PROFILE} --location=$(hostname) &
sleep 20
```

Reply:
Maybe also mention the cores have to be in same node for multiprocessing. Something similar to this:  

Adding the following line to sbatch file may be needed if the user want ensure all the cores are from one node for multiprocessing: `#SBATCH -N 1` and that `#SBATCH --ntasks=20` line is no longer be needed as the whole node will be allocated to the job. Users should be careful while getting complete node allocation as all the memory and cores will be allocated to that job.